content/academy/apify_platform/getting_started/creating_actors.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Creating actors
+description: Build and run your very first actor right on the Apify platform from a template. This lesson provides a hands-on experience with building and running an actor.
+menuWeight: 2
+paths:
+- apify-platform/getting-started/creating-actors
+---
+
+# [](#creating-actors) Creating actors
+
+Creating an actor is so easy to do, we'll show you right in this lesson how to get one up and running in 5 minutes or less. There is no need to install any special software, as everything can be done right on the Apify platform with an Apify account!
+
+## [](#choose-your-template) Choose your template
+
+On the [Apify Console](https://console.apify.com), click on the **Actors** tab we visited in the previous lesson. In the top right-hand corner, you should see a **Create new** button.
+
+![Create new button]({{@asset apify_platform/getting_started/images/create-new-actor.webp}})
+
+Click it, then choose the **Basic Node.js actor** template and click on **Use template**.
```

Comment:
Why did you choose to start with this one and not the **empty** template?

Reply:
don't know actually,  I'll swap it

Reply:
No need to do it now, we'll have to update the code after release of Crawlee and SDK 3 anyway.